### PR TITLE
SOC-1593 | Fetching sorted posts in discussions

### PR DESCRIPTION
**Pull request by Bloodmoon24**
https://wikia-inc.atlassian.net/browse/SOC-1593
@kvas-damian  @rafalkalinski  Please CR

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: front/scripts/main/routes/DiscussionForumRoute.ts [hunk 1]
```diff
@@ -51,7 +50,7 @@ App.DiscussionForumRoute = Em.Route.extend(App.DiscussionLayoutMixin, App.Discus
 		 * @returns {void}
 		 */
 		loadPage(pageNum: number): void {
-			this.modelFor('discussion.forum').loadPage(pageNum);
+			this.modelFor('discussion.forum').loadPage(pageNum, this.controllerFor('discussionForum').get('sortBy'));
```
Comment (rafalkalinski) [inline front/scripts/main/routes/DiscussionForumRoute.ts]: there is defaultSortType property in the DiscussionForumRoute, please use it

**Comment by rafalkalinski** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
Jira ticket is closed, what are you going to do with this PR? Is it waiting for something?

**Comment by kvas-damian** [comment]
Linter check is failing in dev

**Pull request merged**